diff --git a/dwmstatus.c b/dwmstatus.c
index d2a4b03..c5f9170 100644
--- a/dwmstatus.c
+++ b/dwmstatus.c
@@ -17,9 +17,8 @@
 
 #include <X11/Xlib.h>
 
-char *tzargentina = "America/Buenos_Aires";
-char *tzutc = "UTC";
-char *tzberlin = "Europe/Berlin";
+
+char *tzstockholm = "Europe/Stockholm";
 
 static Display *dpy;
 
@@ -155,7 +154,7 @@ getbattery(char *base)
 	} else if(!strncmp(co, "Charging", 8)) {
 		status = '+';
 	} else {
-		status = '?';
+		status = ' ';
 	}
 
 	if (remcap < 0 || descap < 0)
@@ -178,14 +177,9 @@ gettemperature(char *base, char *sensor)
 int
 main(void)
 {
-	char *status;
-	char *avgs;
 	char *bat;
-	char *bat1;
-	char *tmar;
-	char *tmutc;
-	char *tmbln;
-	char *t0, *t1, *t2;
+	char *tmstock;
+	char *status;
 
 	if (!(dpy = XOpenDisplay(NULL))) {
 		fprintf(stderr, "dwmstatus: cannot open display.\n");
@@ -193,30 +187,14 @@ main(void)
 	}
 
 	for (;;sleep(60)) {
-		avgs = loadavg();
 		bat = getbattery("/sys/class/power_supply/BAT0");
-		bat1 = getbattery("/sys/class/power_supply/BAT1");
-		tmar = mktimes("%H:%M", tzargentina);
-		tmutc = mktimes("%H:%M", tzutc);
-		tmbln = mktimes("KW %W %a %d %b %H:%M %Z %Y", tzberlin);
-		t0 = gettemperature("/sys/devices/virtual/hwmon/hwmon0", "temp1_input");
-		t1 = gettemperature("/sys/devices/virtual/hwmon/hwmon2", "temp1_input");
-		t2 = gettemperature("/sys/devices/virtual/hwmon/hwmon4", "temp1_input");
-
-		status = smprintf("T:%s|%s|%s L:%s B:%s|%s A:%s U:%s %s",
-				t0, t1, t2, avgs, bat, bat1, tmar, tmutc,
-				tmbln);
+		tmstock = mktimes("%a %d %b %H:%M | Week %W", tzstockholm);
+
+		status = smprintf("Batte: %s | %s", bat, tmstock);
 		setstatus(status);
 
-		free(t0);
-		free(t1);
-		free(t2);
-		free(avgs);
 		free(bat);
-		free(bat1);
-		free(tmar);
-		free(tmutc);
-		free(tmbln);
+		free(tmstock);
 		free(status);
 	}
 
